package/kernel/linux/modules/leds.mk

```diff
@@ -131,6 +131,22 @@ endef
 $(eval $(call KernelPackage,ledtrig-oneshot))
 
 
+define KernelPackage/leds-apu
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=PC Engines APU1 LED support
+  DEPENDS:= @GPIO_SUPPORT @TARGET_x86
```

Comment:
Any reason for the space after "=" here?

Reply:
Keeping it in same form as above.
https://github.com/openwrt/openwrt/blob/e033e85e30ebf48239b427ce3944786df0bb9ce1/package/kernel/linux/modules/leds.mk#L13